Developer-Guide.md

```diff
@@ -62,6 +63,16 @@ $ sudo kata-runtime kata-check
 
 If your system is *not* able to run Kata Containers, the previous command will error and explain why.
 
+## Disable initrd
+
+Unless you plan to use an initial ramdisk (`initrd` or `initramfs`):
```

Comment:
Suggested rewrite to make this active: "Unless you use an initial ramdisk (`initrd` or `initramfs`):"